frontend/src/components/Quickstart.vue

```diff
@@ -85,7 +85,7 @@ export default defineComponent({
     const introList: IntroItem[] = reactive([
       {
         name: computed(() => t("quick-start.bookmark-an-issue")),
-        link: "/issue/hello-world-101",
+        link: "/issue/hello-world-13001",
```

Comment:
We might want to use different numbers for different builds (dev v.s. release).

https://github.com/bytebase/bytebase/blob/main/store/seed/release/10001__init_seed.sql#L484

Reply:
So for this one, maybe an easy way is to set 13001 to 101 for the test seed data

Reply:
@tianzhou Are you proposing changing this line into 101?

https://github.com/bytebase/bytebase/blob/main/store/seed/test/10130__issue.sql#L17

@d-bytebase WDYT

Reply:
@dragonly yes change 13001 to 101 globally

Reply:
> @dragonly yes change 13001 to 101 globally

LGTM

Reply:
nit: this seems to break the semantic of following ids like 13002-13009

Reply:
yes, I guess for the hello-world, add a comment above should be enough